dist/tools/nacl_key_generator/generate-nacl-keys.c

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
I am not saying that `tweetnacl` might be unsecure, be how we interface to it and the way we use it might be (: